gui/default/index.html

```diff
@@ -474,7 +474,7 @@ <h4 class="panel-title">
                               <span class="fas fa-eye-slash"></span>&nbsp;<span translate>Failed to setup, retrying</span>
                             </span>
                           </div>
-                          <div ng-if="folder.rescanIntervalS <= 0">
+                          <div ng-if="0 >= folder.rescanIntervalS">
```

Comment:
I don't think the syncthing project prefers yoda conditions

Reply:
I fucking hate yoda conditions. I also think it's probably a dead horse, because as long as we use angular with these types of expressions all over the place it's unlikely all future contributors will understand to not use less-than. Worst case they will see some examples like this and think we want yoda everywhere... Iff we wanted to fix this (which I'm not sure it's worth) I guess one way would be to move the expressions into the controller instead...

Reply:
Okay, let's revert this part then.  The other approach would be to wrap it like `!(x >= x)`, but that's even uglier.